### ISSUE TITLE
Exit code 139 on docker

### ISSUE THREAD

**Issue by darkadept** [wekan/wekan#867]
I'm using the docker-compose script at https://github.com/wekan/wekan-mongodb and I'm getting:

`wekan-app exited with code 139`

Docker v1.13.0

**Comment by xet7** [comment]
Works for me, I just tested.
Docker version 1.13.1, build 092cba3.
Backup (info at wiki), cleanup and delete your Wekan/Mongo containers, reinstall, restore.

Closing issue.

**Comment by darkadept** [comment]
This is still happening. I'll document exactly what I'm doing:

Host is Alpine Linux v3.5.1
Docker v1.13.0
Docker-compose v1.11.2

docker-compose.yml file:
```
wekan:
  image: mquandalle/wekan
  links:
    - wekandb
  environment:
    - MONGO_URL=mongodb://wekandb/wekan
    - ROOT_URL=http://localhost
    - MAIL_URL=smtp://myuser:mypass@smtp.myhost.org:25/
    - MAIL_FROM=me@myhost.org
  ports:
    - 8081:80

wekandb:
   image: mongo
   volumes:
     - /srv/wekan/db:/data/db
```

Ran `docker-compose up` to see what's going on and get this output:

```
Creating wekan_wekandb_1
Creating wekan_wekan_1
Attaching to wekan_wekandb_1, wekan_wekan_1
wekandb_1  | 2017-02-22T20:51:49.310+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=ee091167fe8c
  <...more mongodb starting stuff...>
wekandb_1  | 2017-02-22T20:51:49.400+0000 I NETWORK  [thread1] waiting for connections on port 27017
wekan_wekan_1 exited with code 139
```

Docker images are:
```
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
mquandalle/wekan    latest              97dc3cc8dab1        4 hours ago         221 MB
mongo               latest              ad974e767ec4        13 days ago         402 MB
```

I'm not sure why it's exiting with error code 139. Any idea what this is?

**Comment by xet7** [comment]
Yes, here are possibilities:
- Docker is buggy at Alpine Linux. Use some Ubuntu 16.04 based distro.
- Your Alpine Linux has too enough RAM to run Docker.

**Comment by darkadept** [comment]
Ahh! ok. Yes, tried it under a Ubuntu 16.04 distro and it works.

thanks for the info!

**Comment by txomon** [comment]
o/ guys, just in case, while looking for the same error, this looks related to https://github.com/docker/for-linux/issues/58

**Comment by CTimmerman** [comment]
I fixed Docker error 139 in my project by adding a COPY line before my RUN line:

COPY blueprint/src/main/python/setup.py setup.py
RUN python setup.py

**Comment by xet7** [comment]
@replicajune 

Do you mean alpine paxctl fix #1303 that was remove because docker and snap did not build anymore?

**Comment by replicajune** [comment]
hi @xet7,

I didn't knew about paxctl but it seems that would have been a solution. However, grsec kernel hasn't been updated for a year (or so) & I'm not thinking to go back to an obsolete kernel (I used it because it came pre-installed). Besides, the hardened kernel isn't proposed for alpine 3.8 as it was for 3.7 and I don't think it will come back.

I think I'll call it a day, and enjoy wekan with the vanilla kernel :+1:
